src/Experimental/Consolonia.Editor/Resources/SampleFiles/source.js.jsx

```
@@ -0,0 +1,15 @@
+function NumberList(props) {
+    const numbers = props.numbers;
+    const listItems = numbers.map((number) =>
+      <li>{number}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)